jwql/instrument_monitors/common_monitors/bad_pixel_monitor.py

```diff
@@ -1080,11 +1080,9 @@ def process(self, illuminated_raw_files, illuminated_slope_files, flat_file_coun
             if bad_type in FLATS_BAD_PIXEL_TYPES:
                 self.add_bad_pix(bad_location_list, bad_type, illuminated_slope_files,
                                  min_illum_time, mid_illum_time, max_illum_time, baseline_file)
-                flat_png = create_png_from_fits(illuminated_slope_files[0], self.output_dir)
```

Comment:
Removing this line means you can also remove the import (I dont think its actually used at all anymore...maybe comment in there a TODO to consider removing it, or ask @bhilbert4 to create a task to examine its necessity)

Reply:
If we need to still call create_png_ffrom_fits then just delete the `flat_png = `

Reply:
Cool, added it back without the variable assignment.